stactools_sentinel1/stactools/sentinel1/stac.py

```diff
@@ -0,0 +1,172 @@
+            from the Sentinel 1 RTC Geotiff files.
+    """ # noqa
+    product_metadata = RTCMetadata(granule_href)
+
+    item = pystac.Item(id=product_metadata.product_id,
+                       geometry=product_metadata.geometry,
+                       bbox=product_metadata.bbox,
+                       datetime=product_metadata.datetime,
+                       properties={})
+
+    # --Common metadata--
+    # https://github.com/radiantearth/stac-spec/blob/master/item-spec/common-metadata.md
+    item.common_metadata.providers = [
+        c.SENTINEL_PROVIDER, c.SENTINEL_RTC_PROVIDER
+    ]
+
+    if additional_providers is not None:
+        item.common_metadata.providers.extend(additional_providers)
+
+    item.common_metadata.constellation = c.SENTINEL_CONSTELLATION
+    item.common_metadata.platform = product_metadata.platform
+    item.common_metadata.instruments = c.SENTINEL_INSTRUMENTS
+    item.common_metadata.gsd = c.SENTINEL_RTC_SAR['gsd']
+
+    item.common_metadata.start_datetime = product_metadata.start_datetime
+    item.common_metadata.end_datetime = product_metadata.end_datetime
+
+    # STAC Metadata creation date
+    item.common_metadata.created = datetime.utcnow()
+
+    # Addtional properties no belonging to extensions
+    item.properties.update(**product_metadata.metadata_dict)
+
+    # --Extensions--
+    # SAR https://github.com/stac-extensions/sar
+    item.ext.enable('sar')
+    item.ext.sar.frequency_band = c.SENTINEL_FREQUENCY_BAND
+    item.ext.sar.center_frequency = c.SENTINEL_CENTER_FREQUENCY
+    item.ext.sar.observation_direction = c.SENTINEL_OBSERVATION_DIRECTION
+    item.ext.sar.instrument_mode = c.SENTINEL_RTC_SAR['instrument_mode']
+    item.ext.sar.product_type = c.SENTINEL_RTC_SAR['product_type']
+    item.ext.sar.polarizations = c.SENTINEL_RTC_SAR['polarizations']
+    item.ext.sar.resolution_range = c.SENTINEL_RTC_SAR['resolution_range']
+    item.ext.sar.resolution_azimuth = c.SENTINEL_RTC_SAR['resolution_azimuth']
+    item.ext.sar.pixel_spacing_range = c.SENTINEL_RTC_SAR[
+        'pixel_spacing_range']
+    item.ext.sar.pixel_spacing_azimuth = c.SENTINEL_RTC_SAR[
+        'pixel_spacing_azimuth']
+    item.ext.sar.looks_equivalent_number = c.SENTINEL_RTC_SAR[
+        'looks_equivalent_number']
+    item.ext.sar.looks_range = c.SENTINEL_RTC_SAR['looks_range']
+    item.ext.sar.looks_azimuth = c.SENTINEL_RTC_SAR['looks_azimuth']
+
+    # SAT https://github.com/stac-extensions/sat
+    item.ext.enable('sat')
+    item.ext.sat.orbit_state = OrbitState(product_metadata.orbit_state.lower())
+    item.ext.sat.relative_orbit = product_metadata.relative_orbit
+    item.ext.sat.absolute_orbit = product_metadata.absolute_orbit
+
+    # PROJECTION https://github.com/stac-extensions/projection
+    item.ext.enable('projection')
+    item.ext.projection.epsg = product_metadata.epsg
+    item.ext.projection.shape = product_metadata.shape
+    item.ext.projection.proj_bbox = product_metadata.proj_bbox
```

Comment:
```suggestion
    item.ext.projection.bbox = product_metadata.proj_bbox
```

The proj extension has a [`bbox` property](https://github.com/stac-extensions/projection/#item-properties-or-asset-fields) but not a `proj_bbox`. Same is true [in pystac](https://pystac.readthedocs.io/en/latest/api.html#projectionitemext).